webserver/server.py

```diff
@@ -63,6 +64,53 @@ def index():
 
   return render_template("index.html", **context)
 
+@app.route('/recipe_page/<name>')
+def recipe_page(name):
+  context = dict()
+
+  # get recipe data
+  cursor = g.conn.execute("SELECT * from recipes WHERE recipe_name = '" + name.replace("_", " ") + "'")
```

Comment:
We should be concerned about SQL injection right? Might be good to have a helper function that we use to validate strings, but that can be left for a future commit